Anytype/Sources/Helpers/Extensions/AttributedString+Extensions.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension AttributedString {
+    var isEmpty: Bool {
+        startIndex == endIndex
+    }
+}
```

Comment:
We also can check emptiness like this `characters.isEmpty`, in my opinion indexes look a little bit strange here, but maybe I missed something

Reply:
fixed